### PR TITLE
Issue 54

### LINKED ISSUE #54: DeprecationWarning
Hi team,

I got the following warning message while using the endpoints of create template, signature request, and embedded signing.

`(node:1) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.`

Can you please look into the issue? Thank you.

### PR DESCRIPTION
**Pull request by latoyazamill**
Resolving the Buffer() Deprecation Warning Message as presented in GitHub Issue 54.

### CODE REVIEW

**Review by nathanbuchar — COMMENTED** [review]
@latoyazamill I noticed you're using `buffer-from` here. Is there a reason why adding a new dependency here is necessary? Otherwise I'd recommend using the native `Buffer.from()` method: in general it's better to keep Node.js dependencies to a minimum unless a they add functionality that would otherwise be complex or time-consuming to implement and maintain ourselves

I suppose Node version support might be a concern, but `Buffer.from()` has been available since Node v5.10.0 (2016), which is far beyond EOL

**Comment by latoyazamill** [comment]
Hi @nathanbuchar I will test a few things this AM. I chose to include the buffer-from dependency to resolve testing errors that were transpiring when I used Buffer.from(). Example: `Error: expected '\ufffd\ufffdh\ufffd\ufffdZ\ufffd' to equal <Buffer 86 87 68 ae cb 5a ae>`. Any thoughts/suggestions in this case?

**Comment by nathanbuchar** [comment]
Which Node version are you using locally? `node -v`

**Comment by latoyazamill** [comment]
v12.5.0

**Comment by latoyazamill** [comment]
I was able to resolve this with Buffer.from(). Updates coming soon.

File: package.json [hunk 1]
```diff
@@ -13,10 +13,10 @@
   },
   "main": "lib/hellosign.js",
   "dependencies": {
-    "buffer-from": "^1.1.1",
     "expect": "^24.9.0",
     "express": "^4.17.1",
     "form-data": "^2.3.2",
+    "npm": "^6.13.7",
```
Comment (nathanbuchar) [inline package.json]: Whoops! `npm install npm`?

File: test/unit/request_utils.js [hunk 2]
```diff
@@ -69,8 +68,9 @@ describe('Request Utils', function(){
 			var header = hellosign.account._createAuthHeader();
 			var authtype = header.split(' ')[0];
 			var headerAPIKey = header.split(' ')[1];
+      console.log("WHAT IS HEADERAPIKEY? ----->", headerAPIKey);
```
Comment (nathanbuchar) [inline test/unit/request_utils.js]: Stray log

File: test/unit/request_utils.js [hunk 3]
```diff
@@ -90,8 +90,9 @@ describe('Request Utils', function(){
 			var header = hellosignBasic.account._createAuthHeader();
 			var authtype = header.split(' ')[0];
 			var headerAuthString = header.split(' ')[1];
+      console.log("WHAT IS HEADERAUTHSTRING? ---->", headerAuthString);
```
Comment (nathanbuchar) [inline test/unit/request_utils.js]: Stray log

**Review by nathanbuchar — DISMISSED** [review]

**Review by nathanbuchar — COMMENTED** [review]
Tests are failing. Seems like it might be a caching issue on Travis's side, if I had to guess

**Comment by nathanbuchar** [comment]
@latoyazamill can you run:

```
git commit --allow-empty -m "Empty commit"
git push origin issue-54
```

Just go get the tests to rerun. I'd like to do a quick investigation as to why the tests are failing for node v9. I'd guess it's just a trivial issue

**Comment by latoyazamill** [comment]
Yes, I noticed that as well with node v9. I;'d be curious to know what you find in your investigation. I have attempted to push an empty commit and still run into the same errors.

**Comment by nathanbuchar** [comment]
@latoyazamill can you try:

```
rm package-lock.json
npm i
git add package-lock.json
git commit -m "Fix package lock"
git push origin issue-54
```

**Comment by latoyazamill** [comment]
I am doing a two part process in this case because when I attempted to remove and add the package lock, no changes were actually made.

**Comment by nathanbuchar** [comment]
Looks like that did the trick :)

**Review by nathanbuchar — APPROVED** [review]

**Pull request merged**